app/views/layouts/application.html.haml

```diff
@@ -34,12 +34,11 @@
               = render 'application/breadcrumbs'
             = content_for?(:content) ? yield(:content) : yield
         %aside.wrappable__content.promotions
-          %section.promotion{ style: 'margin-top: 20px;'}
-            - if current_site.ad_client.present? && current_site.ad_slot.present?
-              %section.promotion
+          %section.promotion
+            %div.ad
```

Comment:
`.ad` is a component for banner/native ad and some sites use this class to insert ads by js.
So we can't use `.ad` for adsense :scream: